tests/nfv/trex_installation.pm

```diff
@@ -23,24 +19,20 @@ use mmapi;
 use serial_terminal 'select_virtio_console';
 
 sub run {
-    my $trex_repo = "https://github.com/cisco-system-traffic-generator/trex-core.git";
-    my $trex_dest = "/tmp/trex-core";
+    my $trex_version = get_var('TG_VERSION');
+    my $tarball      = "$trex_version.tar.gz";
+    my $url          = "http://trex-tgn.cisco.com/trex/release/$tarball";
+    my $trex_dest    = "/tmp/trex-core";
 
     select_virtio_console();
 
-    zypper_call('--quiet in git-core gcc gcc-c++ make cmake libnuma-devel kernel-source pciutils', timeout => 300);
-
-    # Clone Trex repository
-    assert_script_run("git clone --quiet --depth 1 $trex_repo $trex_dest", timeout => 600);
+    assert_script_run("wget $url");
+    assert_script_run("tar -xzf $tarball");
+    assert_script_run("mv $trex_version $trex_dest");
 
     # Copy sample config file to default localtion
-    assert_script_run("cp $trex_dest/scripts/cfg/simple_cfg.yaml /etc/trex_cfg.yaml");
-
-    # Compile Trex libraries
-    assert_script_run("cd $trex_dest/linux_dpdk");
-    assert_script_run("./b configure", 300);
-    assert_script_run("./b build",     1400);
+    assert_script_run("cp $trex_dest/cfg/simple_cfg.yaml /etc/trex_cfg.yaml");
 }
 
 1;
-
+# vim: set sw=4 et:
```

Comment:
@asmorodskyi has removed these setting from most of the files, I guess now we should avoid it.

Reply:
Removed :)